docs/runtimes/application.ipynb

```diff
@@ -82,16 +82,306 @@
   },
```

Comment:
<h3>Application and serving function integration</h3><p> </p><p>imo a better title</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>